### DISCUSSION TITLE
Found a bug while loading model "tiny.en" and "small.en"

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [openai/whisper#1421] [Q&A] [answered]
I think I found a bug or error while loading these 2 model: "tiny.en" and "small.en"

Here is the code:

```python
import whisper
import traceback

model = "tiny"
language = "en"
audio = "./Recording.wav"

if model != "large" and language == "en":
    model = model + ".en"
    try:
        loaded_model = whisper.load_model(model)
    except Exception as e:
        print(f"Failed to load model: {model}. Exception: {str(e)}")
        traceback.print_exc()
    text = loaded_model.transcribe(audio=audio, language=language)
    print(text["text"])
```

Error
`
Failed to load model: tiny.en. Exception: [WinError 6] The handle is invalid
`
or 
`
Failed to load model: small.en. Exception: [WinError 6] The handle is invalid
`

I have tested base, medium and large model works fine

**Comment by phineas-pta** [comment]
cannot reproduce the error

are you running multiple python sessions ? the error `[WinError 6] The handle is invalid` often involve multiprocessing and/or subprocess

**Reply by rudymohammadbali** [reply]
Nah i'm not running multiple python sessions. But the strange things that is working fine with other models. Am not sure what is the problem.

**Comment by rudymohammadbali** [comment, marked as answer]
The error fixed:

D:\Coding\envs\STT\lib\site-packages\whisper\__init__.py:63: UserWarning: C:\Users\rudyr\.cache\whisper\small.en.pt exists, but the SHA256 checksum does not match; re-downloading the file
